charts/capi-runtime-extensions/templates/clusterclass_nutanix.yaml

```diff
@@ -0,0 +1,6 @@
+# Copyright 2023 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if and .Values.deployDefaultClusterClasses (.Capabilities.APIVersions.Has "infrastructure.cluster.x-k8s.io/v1beta1/NutanixClusterTemplate") }}
```

Comment:
Maybe the changes to pull in the ClusterClass can be its own PR?

Reply:
separated thru https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/473